docs/tutoriales/como-mintear.mdx

```diff
@@ -55,19 +55,19 @@ export const areaxStepper = [
   },
   {
     instruction:
-      "Una vez que haya conectado su billetera, hará clic en Crear NFT. Seleccione el tipo de conjunto que necesita. La cantidad de $ TLOS que necesitará para acuñar dependerá de la cantidad de NFT que vaya a subir.",
+      "Una vez que haya conectado su billetera, haga clic en «Create NFT». Seleccione el tipo de conjunto que necesita. La cantidad de $ TLOS que necesitará para mintear dependerá de la cantidad de NFT que desee subir.",
```

Comment:
What if we stick to Markdown syntax to highlight wathever section we want to the reader?
For example:

`Create NFT` or **Create NFT**

The same thing applies to the rest of the content.